test/validators/no-returns-required/frequency-reported.validator.test.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Thing under test
+const FrequencyReportedValidator = require('../../../app/validators/return-requirements/frequency-reported.validator.js')
+
+describe('Frequency reported validator', () => {
+  describe('when valid data is provided', () => {
+    const payload = {
+      frequencyReported: 'monthly'
+    }
+
+    it('confirms the data is valid', async () => {
+      const result = FrequencyReportedValidator.go(payload)
+
+      expect(result.value.frequencyReported).to.exist()
```

Comment:
How does the property frequencyReported existing confirm that the data is valid? If you add this same expect block to the test below 'when invalid data is provided' the text still passes as frequencyReported still exists. Is there a better way this could be tested?